sdk/core/Azure.Core/api/Azure.Core.net461.cs

```diff
@@ -362,6 +362,12 @@ public abstract partial class ClientOptions
         public static bool operator !=(Azure.Core.ContentType left, Azure.Core.ContentType right) { throw null; }
         public override string ToString() { throw null; }
     }
+    public partial class CoreResponseClassifier : Azure.Core.ResponseClassifier
+    {
+        public CoreResponseClassifier(int[] nonErrors) { }
+        public virtual Azure.Core.CoreResponseClassifier Clone() { throw null; }
```

Comment:
Does Clone need to be public still? i.e. we moved it's usage to Apply

Reply:
Good catch.

Reply:
Updated.